### PR TITLE
Create Model

### PR DESCRIPTION
**Pull request by d2ndjim**
- Created a new rails app.
- Initialize the project with Git.
- Setup Postgres Database
- Created and ran necessary migration files

### CODE REVIEW

File: db/migrate/20220809135529_add_users_ref_to_comments.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```
Comment (AnselemOdims) [inline db/migrate/20220809135529_add_users_ref_to_comments.rb]: - [ ] According to the ERD diagram, the reference is for user table with an alias of author. Please ensure that this is implemented.

File: db/migrate/20220809140045_add_users_ref_to_likes.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```
Comment (AnselemOdims) [inline db/migrate/20220809140045_add_users_ref_to_likes.rb]: - [ ] Please kindly reference above feedback

File: db/migrate/20220809140205_add_users_ref_to_posts.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```
Comment (AnselemOdims) [inline db/migrate/20220809140205_add_users_ref_to_posts.rb]: - [ ] Please kindly reference above feedback

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @d2ndjim , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Lekan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has:

- [ ] Incorrect gitflow. Kindly note that the gitflow of this project should have merge request from **feature-branch into development **.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by d2ndjim** [comment]
Closed

**Pull request closed without merging**